src/patternfly/demos/Card/examples/Card.md

```diff
@@ -319,19 +319,19 @@ import './Card.css'
       {{#> card card--modifier="pf-v6-u-text-align-center"}}
         {{> card-title card-title-text--value="5 Clusters"}}
         {{#> card-body}}
-          <i class="fas fa-check-circle pf-v6-u-success-color-100" aria-hidden="true"></i>
+          <i class="fas fa-check-circle pf-v6-u-icon-color-status-success" aria-hidden="true"></i>
```

Comment:
This works, but IMO ideally these demos should be updated to use the icon component since it has support for status colors.